client-js/app/elements/labrad-registry.html

```diff
@@ -1,30 +1,45 @@
 <link rel="import" href="app-link.html">
-
+<!-- This is the component that simply renders the table of registry content -->
 <dom-module id="labrad-registry">
   <style>
     :host {
       display: block;
     }
+    
+    .list {
+      padding-top: 12px;
+      background-color: white;
+      display: inline-block;
+      @apply(--shadow-elevation-2dp);
+    }    
   </style>
   <template>
+  <div class="list short">
     <table>
+      <thead>
+        <th>Key/Dir Name</th>
+        <th>Key Value</th>
+      </thead>
+      <tbody>
       <template is="dom-repeat" items="{{dirs}}">
-        <tr>
-          <td>
-            <a is="app-link" path="{{item.url}}" href="{{item.url}}">
-              <span class="paper-font-body1">{{item.name}}</span>
-            </a>
-          </td>
-          <td></td>
-        </tr>
-      </template>
-      <template is="dom-repeat" items="{{keys}}">
-        <tr>
-          <td><span class="paper-font-body1">{{item.name}}</span></td>
-          <td><span class="paper-font-body1">{{item.value}}</span></td>
-        </tr>
-      </template>
+          <tr>
+            <td>
+              <a is="app-link" path="{{item.url}}" href="{{item.url}}">
+                <span class="paper-font-body1">{{item.name}}</span>
```

Comment:
Do you want this for this pull request or just in general? I can make a reg-item along the lines of dir-item, or add extra cases to dir-item?